plugins/chat/app/models/chat/channel.rb

```diff
@@ -177,6 +194,7 @@ def self.update_user_counts
             AND (users.suspended_till IS NULL OR users.suspended_till <= CURRENT_TIMESTAMP)
             AND NOT users.staged
             AND user_chat_channel_memberships.following
+            and users.id > 0
```

Comment:
we don't want to take bots into consideration when computing the number of users